### ISSUE TITLE
Latino Model Type

### ISSUE THREAD

**Issue by animate1978** [animate1978/MB-Lab#157]
![latino_model_01](https://user-images.githubusercontent.com/479115/62833684-71695700-bc08-11e9-974e-458481000e03.png)

BlenderArtist [post](https://blenderartists.org/t/mb-lab-release-for-blender-2-80/1138777/293?u=animate1978)

https://vimeo.com/353164612 - Testing the Female Latino model type

So this has become a thing by total accident. Quite a bit needs to be done before this is completed.
***
New Phenotype could be added : South American.
Phenotypes can be edited pretty easy too : https://github.com/animate1978/MB-Lab/blob/master/data/phenotypes/f_as01_ptypes/central_american.json

***
More to come

**Comment by animate1978** [comment]
Since this has been established as a base model, this is being closed. Any further updates or problems should open new issue
